src/main/java/com/skytask/service/ProductService.java

```diff
@@ -0,0 +1,27 @@
+package com.skytask.service;
+
+import com.skytask.model.Product;
+import com.skytask.repository.ProductRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class ProductService {
```

Comment:
Use interfaces to define the Service contracts. Inject interfaces instead of implementations directly.